classes/AppKernel/Arr/IncaParser_Performance.php

```diff
@@ -175,10 +175,12 @@ public function statistics($benchmarkName = NULL, $measurement = NULL)
       return current($this->_benchmarkInfo);
     }  // if ( NULL === $benchmarkName && NULL === $measurement )
 
+        $msg = "Benchmark does not exist: Valid values are ('" .
```

Comment:
I think this code change got put in the wrong place - should be at line 183 and not here.

Reply:
So I pulled this up out of the first `if` statement because it was also used in the second. But now that I'm looking at it again, the array that's being checked in **both** of the `if` statements ( `$this->benchmarkInfo` ) doesn't even exist anymore in the class lol!